services/alerta/service.go

```diff
@@ -100,16 +100,23 @@ func (s *Service) Update(newConfig []interface{}) error {
 }
 
 func (s *Service) Alert(token, resource, event, environment, severity, group, value, message, origin string, service []string, data interface{}) error {
+	c := s.config()
+
 	if resource == "" || event == "" {
 		return errors.New("Resource and Event are required to send an alert")
 	}
 
-	url, post, err := s.preparePost(token, resource, event, environment, severity, group, value, message, origin, service, data)
+	req, err := s.preparePost(token, resource, event, environment, severity, group, value, message, origin, service, data)
 	if err != nil {
 		return err
 	}
 
-	resp, err := http.Post(url, "application/json", post)
+	client := &http.Client{
```

Comment:
Yeah, good catch. I'm pretty sure I understand how the update method works. Let me fix that up.